kit/react-select/index.js

```diff
@@ -5,10 +5,10 @@
  * Copyright © 2019 Extremely Heavy Industries Inc.
  */
 import {elemFactory} from '@xh/hoist/core';
-import Select from 'react-select';
-import AsyncSelect from 'react-select/async';
-import AsyncCreatable from 'react-select/async-creatable';
-import Creatable from 'react-select/creatable';
+import Select from 'react-windowed-select';
+import AsyncSelect from 'react-windowed-select';
```

Comment:
This can all be on one line now

Reply:
Don't we need to export `WindowedSelect` and use that in our `Select` component for us to get any advantage from this change???

As per their readme they re-export all the exports from react-select unmodified, but then also export their custom component. Apologies if I am missing something obvious here 😄